pkg/diskmaker/discovery/discovery.go

```diff
@@ -211,7 +211,7 @@ func ignoreDevices(dev internal.BlockDevice) bool {
 		return true
 	}
 
-	if !(dev.Type == "disk" || dev.Type == "part") {
+	if !(dev.Type == "disk" || dev.Type == "part" || dev.Type == "lvm") {
```

Comment:
it might be cleaner to make a `set.String` and check if `set.Has(dev.Type)`?

Reply:
using `set.String`